drivers/usb/gadget/xaptum/f_psock_gadget.c

```diff
@@ -89,6 +100,76 @@ void f_psock_gadget_work_handler( struct work_struct *work )
  * USB DESCRIPTOR DEFINITIONS
  ***************************************************************************/
 
+/* Control Endpoint Descriptions. 
+ * These descriptors do not vary by speed setting becuase FS, HS and SS all 
+ * have lengths >=64 bytes, and SCM protocol standards limit control 
+ * transfer packets to 64 bytes.
+ */
+static struct usb_endpoint_descriptor
```

Comment:
Control endpoint descriptors. Compare to https://elixir.bootlin.com/linux/latest/source/drivers/usb/gadget/function/f_ecm.c#L192